docs/user-guide/progressive-release/about.md

```diff
@@ -0,0 +1,19 @@
+---
+template: main.html
+---
+
+# About progressive release
+
+Progressive release uses the Iter8 `release` Helm chart. Options are described in the default [values.yaml](https://github.com/iter8-tools/iter8/blob/v0.18.6/charts/release/values.yaml) file. The progressive release tutorials show how it can be used to progressively release versions of an application or ML model.
```

Comment:
I think it would be nice to first give a brief description of what a progressive release is before jumping into the Helm chart and tutorials.